model/src/main/kotlin/config/FileStorageConfiguration.kt

```diff
@@ -19,25 +19,21 @@
 
 package com.here.ort.model.config
 
-import com.fasterxml.jackson.annotation.JsonProperty
+import com.here.ort.utils.storage.FileStorage
+import com.here.ort.utils.storage.HttpFileStorage
+import com.here.ort.utils.storage.LocalFileStorage
 
 /**
- * A class to hold the configuration for using Artifactory as a storage.
+ * The configuration model for a [FileStorage]. Only one of the storage options can be configured.
```

Comment:
consider adding a require for: 'Only one of the storage options can be configured.'
or alternatively an enabled flag

Reply:
For now I keep this check in `ScanResultsStorage` as it was before, but I agree validation of the configuration model should be improved. I am thinking about adding a dedicated validation function to handle this later on.

Reply:
ok, great! thanks!